ui/src/app/views/signed-in/component.ts

```diff
@@ -83,6 +86,18 @@ export class SignedInComponent implements OnInit {
         this.navigateSignOut();
       }
     });
+
+    this.profileStorageService.profile$.subscribe((profile) => {
```

Comment:
:point_up:

Reply:
@calbach and @jacmrob - please see https://github.com/all-of-us/workbench/pull/1468 ... I think this is what CH is asking for. I took it a little further since there are actually two profile subscriptions there.